Sources/SwiftDocC/Semantics/Metadata/Availability.swift

```diff
@@ -0,0 +1,138 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import Markdown
+
+/// A directive that sets the platform availability information for a documentation page.
+///
+/// `@Available` is analagous to the `@available` attribute in Swift: It allows you to specify a
+/// platform version that the page relates to. To specify a platform and version, list the platform
+/// name and use the `introduced` argument:
+///
+/// ```markdown
+/// @Available(macOS, introduced: "12.0")
+/// ```
+///
+/// The available platforms are `macOS`, `iOS`, `watchOS`, and `tvOS`.
+///
+/// This directive is available on both articles and documentation extension files. In extension
+/// files, the information overrides any information from the symbol itself.
+///
+/// This directive is only valid within a ``Metadata`` directive:
```

Comment:
Maybe we should have an example in the docs here of using multiple availability directives? It wasn't obvious to me that, that was supported until I read the code.